baml_language/sdks/csharp/bridge_csharp/tests/Baml.Bridge.GenericCompileProbe/verify.sh

```
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly script_dir="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)